src/definitions/definitionBlocks.ts

```diff
@@ -156,17 +152,19 @@ export class OutputDefinitionBlock<TypeName extends string> {
     typeName: BaseScalars,
     opts: [] | ScalarOutSpread<TypeName, any>
   ) {
-    let config: NexusOutputFieldDef = {
-      name: fieldName,
-      type: typeName,
-    }
     if (typeof opts[0] === 'function') {
-      // FIXME ditto to the one in `field` method
-      config.resolve = opts[0] as any
-    } else {
-      config = { ...config, ...opts[0] }
+      console.warn(
```

Comment:
Should this be a `console.error` rather than a warn? If we're going to gracefully degrade (which I don't think is a bad option, to remove the types but support the existing runtime behavior) then I think it'd make sense as a `warn`. Otherwise I think it should be a `console.error`.

Reply:
> (which I don't think is a bad option, to remove the types but support the existing runtime behavior)

Ah that's a great point, sounds good to me. I'll mention in the log that this will be a hard runtime error in the next release.